tests/test_ts_group.py

```diff
@@ -752,4 +754,107 @@ def test_getitem_attribute_error(self, ts_group):
     )
     def test_getitem_boolean_fail(self, ts_group, bool_idx, expectation):
         with expectation:
-            out = ts_group[bool_idx]
\ No newline at end of file
+            out = ts_group[bool_idx]
+
+    def test_merge_complete(self, ts_group):
+        with pytest.raises(TypeError) as e_info:
+            nap.TsGroup.merge_group(ts_group, str, dict)
+            assert str(e_info.value) == f"Input at positions {[2, 3]} are not TsGroup!"
```

Comment:
```suggestion
        with pytest.raises(TypeError,  match=f"Input at positions {[2, 3]} are not TsGroup!"):
            nap.TsGroup.merge_group(ts_group, str, dict)
```

Reply:
Interestingly, this change gave a perplexing error

```
E           AssertionError: Regex pattern did not match.
E            Regex: 'Input at positions [2, 3] are not TsGroup!'
E            Input: 'Input at positions [2, 3] are not TsGroup!'
E            Did you mean to `re.escape()` the regex?
```

Reply:
Fixed with replacing `{[2, 3]}` with `(.*)`